lib/erblint-github/linters/github/accessibility/details_has_summary.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative "../../custom_helpers"
+
+module ERBLint
+  module Linters
+    module GitHub
+      module Accessibility
+        class DetailsHasSummary < Linter
+          include ERBLint::Linters::CustomHelpers
+          include LinterRegistry
+
+          MESSAGE = "<details> elements need to have explict <summary> elements"
+
+          def run(processed_source)
+            tags(processed_source).each do |tag|
```

Comment:
```suggestion
            tags(processed_source).each_with_index do |tag, index|
```
then you could get rid of line 21!